src/Evolution/DiscontinuousGalerkin/Limiters/Minmod.tpp

```diff
@@ -122,7 +122,7 @@ void Minmod<VolumeDim, tmpl::list<Tags...>>::pup(PUP::er& p) noexcept {
 
 template <size_t VolumeDim, typename... Tags>
 void Minmod<VolumeDim, tmpl::list<Tags...>>::package_data(
-    const gsl::not_null<PackagedData*>& packaged_data,
+    gsl::not_null<PackagedData*> packaged_data,
```

Comment:
okay, it should still be const here then shouldn't it?

Reply:
It should 👍, I'll squash in this change momentarily